mmv1/products/networksecurity/UrlLists.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Resource
+name: "UrlLists"
+base_url: "projects/{{project}}/locations/{{location}}/urlLists"
+create_url: "projects/{{project}}/locations/{{location}}/urlLists?urlListId={{name}}"
+self_link: "projects/{{project}}/locations/{{location}}/urlLists/{{name}}"
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  UrlList proto helps users to set reusable, independently manageable lists of hosts, host patterns, URLs, URL patterns.
+references:
+  !ruby/object:Api::Resource::ReferenceLinks # TODO(diogoesteves): change the url to beta once it is available.
```

Comment:
Can you please add the guide https://cloud.google.com/secure-web-proxy/docs/use-url-list? Thanks.

Reply:
Hi, added the field for guides and used the one you pointed.